db/migrate/20151013181808_change_item_duedate_to_date.rb

```diff
@@ -0,0 +1,8 @@
+class ChangeItemDuedateToDate < ActiveRecord::Migration
+  def up
+    execute 'ALTER TABLE items ALTER due_date TYPE date USING (due_date::date)'
+  end
+  def down
+    execute 'ALTER TABLE items ALTER due_date TYPE date USING (due_date::string)'
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.